### PR TITLE
Add draw script

### PR DESCRIPTION
**Pull request by mathieu-lemay**

### CODE REVIEW

File: draw.py [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+
+
+standings = _get_standings(2021)
+
+for s in standings:
+    print(f"{s.position:-2d}. {s.name}: {s.draw_entries}")
+
+
+coupons = []
+exclusions = ["William Cantin"]
```
Comment (mathieu-lemay) [inline draw.py:166]: Doesn't work here anymore.

File: draw.py [hunk 2]
```diff
@@ -0,0 +1,185 @@
...
+random.shuffle(coupons)
+
+input("\nPress enter to draw the first winner")
+winner = draw(coupons)
+
+coupons = [c for c in coupons if c != winner]
+
+input("\nPress enter to draw the second winner")
+winner = draw(coupons)
+coupons = [c for c in coupons if c != winner]
```
Comment (mvives-broadsign) [inline draw.py:182]: Right now the first winner can win at the third draw I think, should we exclude that ?

**Review by mvives-broadsign — CHANGES_REQUESTED** [review]

File: draw.py (hunk 2 above)
Comment (mathieu-lemay) [reply draw.py:182]: I overwrite the coupons every time, so at this point, the first winner is already not in there anymore.

File: draw.py (hunk 1 above)
Comment (wilomgfx) [reply draw.py:166]: Savage 😭

Comment (mathieu-lemay) [reply draw.py:166]: lol, sorry buddy, nothing personal! It would be a bit awkward to draw your name in the all hands this week.

Comment (wilomgfx) [reply draw.py:166]: For sure hahaha!

Just messing with you 😈

**Pull request merged**